framework/include/mfem/auxkernels/MFEMComplexInnerProductAux.h

```diff
@@ -0,0 +1,63 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifdef MOOSE_MFEM_ENABLED
+#pragma once
+
+#include "MFEMComplexAuxKernel.h"
+#include "mfem.hpp"
+
+/**
+ * Project s * (U \cdot V*) onto a vector MFEM auxvariable.
+ *
+ * Notes:
+ *   - Currently supports only interior DOFs (no shared/constrained DOFs).
+ *   - Enforces 3D: mesh dimension and all involved vdim must be 3.
+ *   - Takes into account Hermitian conjugation when computing the dot product.
+ */
```

Comment:
Edit as per `MFEMInnerProductAux.h` for correct math formatting and updated notes.